apps/models/app_types/custom/gradio.py

```diff
@@ -0,0 +1,21 @@
+from ... import AppInstanceManager, BaseAppInstance
+from .base import AbstractCustomAppInstance
+
+
+class GradioAppInstanceManager(AppInstanceManager):
+    model_type = "gradioappinstance"
+
+
+class GradioInstance(AbstractCustomAppInstance, BaseAppInstance):
+    objects = GradioAppInstanceManager()
+
+    def get_k8s_values(self):
+        k8s_values = super().get_k8s_values()
+        # TODO Change this to actual command to run gradio app
+        k8s_values["appconfig"]["startupCommand"] = "python /app.py"
```

Comment:
Is there a reason we are using /app.py here? Maybe this could be
```suggestion
        k8s_values["appconfig"]["startupCommand"] = "python app.py"
```

Also, this probably means that the gradio filename can only be app.py?